apis/r/tests/testthat/test-shape.R

```diff
@@ -566,7 +566,9 @@ test_that("SOMADenseNDArray shape", {
       expect_true(all(readback_shape == readback_maxshape))
     }
 
-    expect_true(all(readback_shape == readback_maxshape))
+    if (! tiledbsoma:::.dense_arrays_can_have_current_domain()) {
+      expect_true(all(readback_shape == readback_maxshape))
```

Comment:
`expect_equal()` yields a more informative error message should this test fail
```R
> readback_shape <- c(80L, 230L)
> readback_maxshape <- c(240L, 30L)
> testthat::expect_equal(readback_shape, readback_maxshape)
Error: `readback_shape` not equal to `readback_maxshape`.
2/2 mismatches (average diff: 115)
[1]  80 - 240 == -160
[2] 230 - 300 ==  -70
Execution halted
> testthat::expect_true(all(readback_shape == readback_maxshape))
Error: all(readback_shape == readback_maxshape) is not TRUE

`actual`:   FALSE
`expected`: TRUE
Execution halted
```
```suggestion
      expect_equal(readback_shape, readback_maxshape)
```